client/src/partials/reports/global_transaction/global_transaction.html

```diff
@@ -107,10 +100,9 @@
                 <td width="20%"> {{ record.trans_id }} </td>
```

Comment:
20% + 20% + 20% + 20% + 20% + 60% = 160%. Please correct your widths so that the text is visible!